go/pkg/cas/upload_test.go

```diff
@@ -205,8 +224,13 @@ func inputChanFrom(inputs ...*UploadInput) chan *UploadInput {
 type fakeCAS struct {
```

Comment:
Can we use https://pkg.go.dev/github.com/bazelbuild/remote-apis-sdks@v0.0.0-20210423213014-847bca232f88/go/pkg/fakes#CAS instead?

Reply:
I considered that, but I am not sure it would save a lot of code, and I am not sure how would I check exactly what requests were sent, on how would I return transient values. On the other hand, `fastCAS` seems pretty small.

Reply:
I see.